ExternalDNS Operator update to release notes OSDOCS-12457 (#84230)
openshift/openshift-docs: networking/external_dns_operator/external-dns-operator-release-notes.adoc

[id="external-dns-operator-release-notes-1.3.0"]
== External DNS Operator 1.3.0

The following advisory is available for the External DNS Operator version 1.3.0:

*link:https://access.redhat.com/errata/RHEA-2024:8550[RHEA-2024:8550 Produce Enhancement Advisory]

This update includes a rebase to the 0.14.2 version of the upstream project:

> * link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/[Kubernetes 0.14.2 Information]

Reviewer: This link still opens to wrong page `https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/`

Reviewer 2: Right. I don't think we need a link to any upstream at all. We try to stay away from referencing upstream as it has a broader set of features from what we support.

Author: okay, will remove link.

[id="external-dns-operator-.3.0-bug-fixes"]
=== Bug fixes

> Previously, the ExternalDNS Operator would not deploy on HCP clusters. With this release, they deploy in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])

Reviewer 3: ```suggestion
Previously, the ExternalDNS Operator would not deploy on HCP clusters. With this release, the Operator deploys in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])
```

Author: Thank you

> Previously, the ExternalDNS Operator would not deploy on HCP clusters. With this release, they deploy in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])

Reviewer 2: ```suggestion
Previously, the ExternalDNS Operator could not deploy operands on HCP clusters. With this release, the operator deploys operands in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])
```

Author: Fixed per Melvin's comment

> Previously, the ExternalDNS Operator would not deploy on HCP clusters. With this release, they deploy in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])

Author: Thank you

> Previously, the ExternalDNS Operator was not using RHEL 9 as its building or base images.With this release, RHEL9 is the base. (link:https://issues.redhat.com/browse/OCPBUGS-41683[*OCPBUGS-41683*])

Reviewer 3: ```suggestion
Previously, the ExternalDNS Operator was not using RHEL 9 as its building or base images. With this release, RHEL9 is the base image. (link:https://issues.redhat.com/browse/OCPBUGS-41683[*OCPBUGS-41683*])
```

> Previously, the godoc had a broken link. With this release, the godoc uses a GitHub permalink. (link:https://issues.redhat.com/browse/OCPBUGS-36797[*OCPBUGS-36797*])

Reviewer 2: ```suggestion
Previously, the godoc had a broken link for Infoblox provider. With this release, the godoc has been reviewed and some links were removed while some other were replaced with GitHub permalinks. (link:https://issues.redhat.com/browse/OCPBUGS-36797[*OCPBUGS-36797*])
```

Author: ty

[id="external-dns-operator-release-notes-1.3.0"]
== External DNS Operator 1.3.0

The following advisory is available for the External DNS Operator version 1.3.0:

*link:https://access.redhat.com/errata/RHEA-2024:8550[RHEA-2024:8550 Produce Enhancement Advisory]

This update includes a rebase to the 0.14.2 version of the upstream project.

[id="external-dns-operator-.3.0-bug-fixes"]
=== Bug fixes

> Previously, the ExternalDNS operand would not be deployable on HCP clusters. With this release, the Operand deploys in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])

Reviewer: @jmanthei I think wordings of Andrey is right.
`Previously, the ExternalDNS Operator could not deploy operands on HCP clusters. With this release, the operator deploys operands in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])`

Author: okay, no problem. I will update.

> Previously, the ExternalDNS Operator could not deploy operands on HCP clusters. With this release, the operator deploys operands in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])

Author: operator capitalized

> Previously, the ExternalDNS Operator was not using RHEL 9 as its building or base images.With this release, RHEL9 is the base. (link:https://issues.redhat.com/browse/OCPBUGS-41683[*OCPBUGS-41683*])

Reviewer 3: Did my comment from yesterday get lost?
Is this right? The base image is a different name for the building image?
```suggestion
Previously, the ExternalDNS Operator was not using RHEL 9 as its building, or base, image. With this release, RHEL9 is the base image. (link:https://issues.redhat.com/browse/OCPBUGS-41683[*OCPBUGS-41683*])
```

> Previously, the ExternalDNS Operator was not using RHEL 9 as its building or base images.With this release, RHEL9 is the base. (link:https://issues.redhat.com/browse/OCPBUGS-41683[*OCPBUGS-41683*])

Author: I believe building and base images are separate things...
